content/applications/inventory_and_mrp/manufacturing/quality_control/quality_checks.rst

```diff
@@ -0,0 +1,131 @@
+regular intervals. When quality checks are created by a |QCP|, they appear on a manufacturing or
+inventory order, where the employee processing the order will be prompted to complete them. For a
+full explanation of how to create and configure a |QCP|, see the documentation on :ref:`Quality
+Control Points <manufacturing/quality_control/quality-control-points>`.
+
+While quality checks are most commonly created automatically by a |QCP|, it is also possible to
+manually create a single quality check. Creating a check manually is useful when an employee wants
+to schedule a quality check that will only occur once, or register a quality check that they conduct
+unprompted.
+
+Manual quality check
+====================
+
+To manually create a single quality check, navigate to :menuselection:`Quality --> Quality Control
+--> Quality Checks`, and click :guilabel:`New`. On the quality check form, begin by selecting an
+option from the :guilabel:`Control per` drop-down menu:
+
+- :guilabel:`Operation` requests a check for an entire operation (ex. delivery order) and all
+  products within it.
+- :guilabel:`Product` requests a check for every unit of a product that is part of an operation (ex.
+  every unit of a product within a delivery order).
+- :guilabel:`Quantity` requests a check for every quantity of a product that is part of an operation
+  (ex. one check for five units of a product within a delivery order). Selecting
+  :guilabel:`Quantity` also causes a :guilabel:`Lot/Serial` drop-down field to appear, from which
+  can be selected a specific lot or serial number that the quality check should be conducted for.
+
+Next, select an inventory operation from the :guilabel:`Picking` drop-down menu or a manufacturing
+order from the :guilabel:`Production Order` drop-down menu. This is necessary because Odoo needs to
+know for which operation the quality check is being conducted.
+
+If the quality check should be assigned to a specific |QCP|, select it from the :guilabel:`Control
+Point` drop-down menu. This is useful if the quality check is being created manually, but should
+still be recognized as belonging to a specific |QCP|.
+
+Select a quality check type from the :guilabel:`Type` drop-down field:
+
+- :guilabel:`Instructions` provides specific instructions for how to conduct the quality check.
+- :guilabel:`Take a Picture` requires a picture to be attached to the check before the check can be
+  completed.
+- :guilabel:`Pass - Fail` is used when the product being checked must meet a certain criteria to
+  pass the check.
+- Selecting :guilabel:`Measure` causes a :guilabel:`Measure` input field to appear, in which a
+  measurement must be entered before the check can be completed.
+- Selecting :guilabel:`Worksheet` causes a :guilabel:`Quality Template` drop-down field to appear,
+  from which can be selected a quality worksheet that must be filled out to complete the check.
+
```

Comment:
Leaving a mental note to add a seealso admonition block to the Instructions and Measure docs you're preparing! (: